deliver/lib/deliver/runner.rb

```diff
@@ -50,6 +51,11 @@ def upload_metadata
       # First, collect all the things for the HTML Report
       screenshots = UploadScreenshots.new.collect_screenshots(options)
       UploadMetadata.new.load_from_filesystem(options)
+
+      # Normalizes languages keys from symbols to strings
+      UploadMetadata.new.normalize_language_key(options)
```

Comment:
It DOES! I kind of wanted in both places for a really stupid reason (cause made tests make more sense) but will look into doing these in a different place